tests/integration/targets/azure_rm_cosmosdbaccount/tasks/main.yml

```diff
@@ -0,0 +1,106 @@
+- name: Prepare random number
+  set_fact:
+    rpfx: "{{ resource_group | hash('md5') | truncate(7, True, '') }}{{ 1000 | random }}"
+  run_once: yes
+
+- name: Create instance of Database Account -- check mode
+  azure_rm_cosmosdbaccount:
+    resource_group: "{{ resource_group }}"
+    name: "cosmos{{ rpfx }}"
```

Comment:
why not make  name a variable, each test need to understand how to concat the name string